tests/components/min_max/test_sensor.py

```diff
@@ -168,6 +174,35 @@ def test_mean_4_digit_sensor(self):
         assert entity_ids[2] == state.attributes.get("min_entity_id")
         assert self.max == state.attributes.get("max_value")
         assert entity_ids[1] == state.attributes.get("max_entity_id")
+        assert self.median == state.attributes.get("median")
+
+    def test_median_sensor(self):
+        """Test the median sensor."""
+        config = {
+            "sensor": {
+                "platform": "min_max",
+                "name": "test_median",
+                "type": "median",
+                "entity_ids": ["sensor.test_1", "sensor.test_2", "sensor.test_3"],
+            }
+        }
+
+        assert setup_component(self.hass, "sensor", config)
+
+        entity_ids = config["sensor"]["entity_ids"]
+
+        for entity_id, value in dict(zip(entity_ids, self.values)).items():
+            self.hass.states.set(entity_id, value)
+            self.hass.block_till_done()
+
+        state = self.hass.states.get("sensor.test_median")
+
+        assert str(float(self.median)) == state.state
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5872960)